config/routes.rb

```diff
@@ -47,7 +47,7 @@
 
   get 'signup/:invitation_token' => 'users#new', as: 'signup'
   get 'claim/:invitation_token' => 'external_authors#claim', as: 'claim'
-  get 'complete_claim/:invitation_token' => 'external_authors#complete_claim', as: 'complete_claim'
+  post 'complete_claim/:invitation_token' => 'external_authors#complete_claim', as: 'complete_claim'
```

Comment:
The problem was:
```
No route matches [POST] "/complete_claim/4fbc1797f6db942c2d4ebbd8958c39fec8a1c47a" (ActionController::RoutingError)
```
Refer to "Route Matching" on https://www.sitepoint.com/get-your-app-ready-for-rails-4/ -- basically, you need to specify the correct verb now